sys/include/net/gcoap.h

```diff
@@ -241,6 +241,16 @@ extern "C" {
 #define GCOAP_PAYLOAD_MARKER    (0xFF)
 
 /**
+ * @name    Possible request codes
+ * @{
+ */
+#define GCOAP_GET               (1) /**< GET request */
```

Comment:
oh, but I just see, that nanocoap also defines:
```
#define COAP_CLASS_REQ          (0)
#define COAP_METHOD_GET         (1)
#define COAP_METHOD_POST        (2)
#define COAP_METHOD_PUT         (3)
#define COAP_METHOD_DELETE      (4)
```
So these should do the trick, though very confusing from a user point of view...